test/libyul/yulOptimizerTests/expressionSplitter/control_flow.yul

```diff
@@ -1,7 +1,8 @@
 {
 	let x := calldataload(0)
 	if mul(add(x, 2), 3) {
-		for { let a := 2 } lt(a, mload(a)) { a := add(a, mul(a, 2)) } {
```

Comment:
I think this test does not have to be modified.

Reply:
Yeah forgot that this doesn't apply to loop conditions